FogCreek/Glitch-Community
# Cooked devourer

## Links
* cooked-devourer.glitch.me/pupdates/preview
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/196

## GIF/Screenshots:
![Screen Shot 2019-07-29 at 4 54 46 PM](https://user-images.githubusercontent.com/3011231/62081895-97165980-b221-11e9-93f0-cd8ef1a4cf13.png)

## Changes:
* Loads pupdates from a JSON file
* added preview page
* on preview page, reads JSON from another project (currently `buttercup-room`) to preview additions/changes to pupdates
* on hitting publish, will update the pupdates JSON in this remix

## How To Test:
* log into buttercup-room.glitch.me
* make some changes
* check them out on cooked-devourer.glitch.me/pupdates/preview
* log into cooked-devourer remix
* hit the publish button
* check that your updates are in the pupdates on that remix (may need to run `window.localStorage["community-userPrefs"] = {"newStuffReadId":0}` in the console to reset your pupdate read count)

## Feedback I'm looking for:
Does it work as expected? is it confusing? etc?

## Things left to do before deploying:
- Get approval on buttercup-room work as well
- change project names to something memorable
- update references in both projects to correct remix names
